src/lib.rs

```diff
@@ -76,6 +76,9 @@ pub fn read_ase<T: std::io::Read>(mut ase: T) -> Result<(Vec<Group>, Vec<ColorBl
     let mut color_blocks = Vec::new();
     let mut buf_u16 = [0; 2];
 
+    let mut group_hld = types::GroupHold::Empty;
+    let mut group_hld_value = Group::default();
```

Comment:
```suggestion
    let mut group_hold = types::GroupHold::Empty;
    let mut group_hold_value = Group::default();
```
Two characters won't kill us, but they make reading easier :)